plugins/webp-uploads/settings.php

```diff
@@ -121,6 +152,21 @@ function webp_uploads_generate_webp_jpeg_setting_callback(): void {
 	<?php
 }
 
+/**
+ * Renders the settings field for the 'webp_uploads_use_picture_element' setting.
+ *
+ * @since 1.0.0
+ */
+function webp_uploads_use_picture_element_callback(): void {
+	?>
+			<label for="webp_uploads_use_picture_element">
+			<input name="webp_uploads_use_picture_element" type="checkbox" id="webp_uploads_use_picture_element" aria-describedby="webp_uploads_use_picture_element_description" value="1"<?php checked( webp_uploads_is_picture_element_enabled() ); ?> />
+			<?php esc_html_e( 'Use <picture> Element', 'webp-uploads' ); ?>
```

Comment:
With https://github.com/WordPress/performance/pull/73/files#r1619500444 and:

```suggestion
			<?php esc_html_e( 'Use <picture> Element', 'webp-uploads' ); ?>
			<em><?php esc_html_e( '(experimental)', 'webp-uploads' ); ?></em>
```

Before | After
--|--
![Screenshot 2024-05-29 14 31 56](https://github.com/WordPress/performance/assets/134745/aa3c94a2-2107-4c73-af48-6ff5633a3fe2) | ![Screenshot 2024-05-29 14 34 56](https://github.com/WordPress/performance/assets/134745/473a7a88-1e1f-46c4-867f-aacdbf9f2ebb)